docs/sources/tempo/release-notes/v2-6.md

```diff
@@ -168,7 +168,11 @@ vParquet4 format is now the default block format.
 It's production ready and we highly recommend switching to it for improved query performance. [PR [3810](https://github.com/grafana/tempo/pull/3810)]
 
 Upgrading to Tempo 2.6 modifies the Parquet block format.
-Although you can use Tempo 2.6 with vParquet2 or vParquet3, you can only use Tempo 2.6 with vParquet3.
+You don't need to do anything with Parquet to go from 2.5 to 2.6.
+All of your old blocks remain vParquet3, which 2.6 reads.
+Tempo 2.6 creates vParquet4 blocks by default, which enables the new TraceQL features.
+
+Although you can use Tempo 2.6 with vParquet2 or vParquet3, you can only use vParquet4 with Tempo 2.6 and newer.
```

Comment:
vParquet4 was already available as an "experimental" block format in 2.5 and became the default in 2.6 so users can already use vParquet4 in Tempo 2.5 - but they would need to upgrade to 2.6 to get all the new traceql features

Reply:
With the other update, we can remove part of this sentence. 
```suggestion
Although you can use Tempo 2.6 with vParquet2 or vParquet3, you can only use vParquet4 with Tempo 2.5 and newer.
If you are using 2.5 with vParquet4, you'll need to upgrade to Tempo 2.6 to use the new TraceQL features. 
```